packages/core/src/components/overflow-ellipsis/overflowEllipsis.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { IProps } from "../../common/props";
+
+export interface IOverflowEllipsisState {
+    content?: string;
```

Comment:
Why don't you just make it an empty string exactly where you're setting it right now (`public state ...`)